rock-paper-scissors.js

```diff
@@ -56,6 +55,12 @@ function game() {
   let computerScore = 0
   for (let i = 0; i < 5; i++) {
     let playerSelection = prompt ("Pick a hand");
+    playerSelection = playerSelection.toLowerCase();
+    if (playerSelection != "rock" && playerSelection != "paper" && playerSelection != "scissors"){
```

Comment:
This can be refactored to look like this:
```
const validInputs = ["rock", "paper", "scissors"];
if (validInputs.includes("playerSelection"));
```